src/examples/px4_simple_app/px4_simple_app.c

```diff
@@ -38,92 +38,12 @@
  * @author Example User <mail@example.com>
  */
 
-#include <px4_config.h>
-#include <px4_tasks.h>
-#include <px4_posix.h>
-#include <unistd.h>
-#include <stdio.h>
-#include <poll.h>
-#include <string.h>
-#include <math.h>
-
-#include <uORB/uORB.h>
-#include <uORB/topics/sensor_combined.h>
-#include <uORB/topics/vehicle_attitude.h>
+#include <px4_log.h>
 
 __EXPORT int px4_simple_app_main(int argc, char *argv[]);
 
 int px4_simple_app_main(int argc, char *argv[])
 {
-	PX4_INFO("Hello Sky!");
```

Comment:
Unintentional change?

Reply:
Yes probably. I only want to commit the Airframe file + the 2 mixer files. Anything else I've changed is irrelevant